src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
+@import './blocks/variables';
+
 body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  width: $calendar-size;
+
+  padding: $calendar-padding;
+
+  &__day {
+    box-sizing: border-box;
+    width: $day-size;
+    height: $day-size;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    border: 1px solid #000;
+
+    background-color: #eee;
+    transition: all;
```

Comment:
It is recommended to specify the exact properties instead of 'all' for better performance